### ISSUE TITLE
Additional Divider Options: Curse, Trash, Starting Deck

### ISSUE THREAD

**Issue by Zannick** [sumpfork/dominiontabs#175]
My current organization uses a Curse divider for 10 cards apiece so I don't have to do any counting at setup, so it would be great if there were an option to add more Curse dividers to the pdf (as opposed to reprinting a page).

I also keep Starting Decks separately, and thanks to having Dominion/Intrigue I can have up to 8 of these.

Additionally, I don't care about the Trash dividers that show up for 1st Edition Dominion/Intrigue.

In short, I'm suggesting a "Starting Deck" base divider plus the ability to tweak the numbers of base dividers.

**Comment by wvoigt** [comment]
@Zannick 
I am working on options for `--no-trash`, `--curse10`, `--start-decks`.
I need to figure out how to represent the start deck dividers and the extra curse dividers on the expansion dividers.  It looks a little odd alphabetically with the extra entries.

**Comment by sumpfork** [comment]
Done with #201, release pending.
